x/oracle/client/cli/tx.go

```diff
@@ -0,0 +1,28 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+var (
+	DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
```

Comment:
What's this? Is it used?

Reply:
Nope, got this from a starport tutorial project - I'll take it out